Automattic/jetpack
# Publicize: Add the connections post field

#### Changes proposed in this Pull Request:

This adds the REST post field to the Publicize package, and configures in via the setup. It's adapted version of WPCOM_REST_API_V2_Post_Publicize_Connections_Field removing the dependency on WPCOM_REST_API_V2_Field_Controller.

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:
This can be manually tested by using the block editor to publish posts, and checking that Publicize still works as expected.

The automated tests can be run by running `composer phpunit` after cd-ing into the package's directory.
